### ISSUE TITLE
The future of package

### ISSUE THREAD

**Issue by blackfriar** [AudithSoftworks/Uniform#411]
This forum is dead.

**Comment by shehi** [comment]
@fidian, how far have you gone with your code-rewriting?

**Comment by fidian** [comment]
Everything I wrote was committed to the v3 branch.  It mostly works and needs some finishing and cross-browser testing.  What I would love to have is a way to test in multiple browsers automatically.  I also started work on rebuilding documentation and sprite files automatically from individual images.  I even cleaned up the CSS to make it less complex and then made the sprite images have less duplication.  If you have questions about my commits, feel free to email me directly - fidian@rumkin.com.

**Comment by shehi** [comment]
@fidian When I do my front-end work nowadays, I generally follow the rule "support last 2-3 versions of browsers only". That makes things easier to progress, especially when you are developing an open source project with limited resources and want things to actually move ahead. And with that, it becomes much easier to follow latest developments in front-end arena, specifically CSS-3. Everytime I see this sprite image in this project, I can't help but think that most of the effects could easily been achieved via ```*-gradient``` and ```*-shadow``` properties in CSS-3.

Long story short, what you think if we limited our future major releases (as in v3) with preset policies like this and only provide bugfix support to legacy releases as we move forward?

**Comment by fidian** [comment]
I'm really not involved in this project - my last commit was years ago, if I remember right.  I don't know how much my vote should count.

That said, it is normal for software to start to use new features and progress with the times.  If there's a rewrite coming, it would make sense to target a new baseline for browsers and minimum feature set.

One thing about the gradient is that it wouldn't work out well for people who have complicated themes.  Let's pretend that I wanted a wooden picture frame with some striped and textured design going through the input field (an actual example I've seen).  Gradients would not cut it but the sprite images could be used to achieve that effect.

**Comment by shehi** [comment]
@pixelmatrix , @fidian : would it be accurate to assume the folders /www and /www-fragments are there only for documentation purposes and can actually be moved to the site?

**Comment by fidian** [comment]
That's exactly how I remember those two folders.

**Comment by shehi** [comment]
Ok folks:
# Site is up
# Repo is moved to its new location
# Repo added to both Bower and NPM

In coming days I will be planning on roadmap for 3.0 and remaining bugfixes. For the moment, closing this one.
